gatsby-config.ts

```diff
@@ -249,13 +250,6 @@ const config: GatsbyConfig = {
                   },
                   custom_elements: [
                     {
-                      'media:content': {
-                        _attr: {
-                          height: 300,
-                          medium: 'image',
-                          url: imageUrl
-                        }
-                      },
```

Comment:
I'm not sure if you're not removing the default config of height 300px for images here. You'd have to check that manually against the current deploy.